back/engines/free/document_annotation/app/serializers/document_annotation/web_api/v1/document_annotation_participation_context_serializer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module DocumentAnnotation::WebApi::V1::DocumentAnnotationParticipationContextSerializer
+  extend ActiveSupport::Concern
+
+  included do
+    with_options if: proc { |object|
+      object.participation_context?
+    } do
```

Comment:
Just a small optional shortcut :) 

```suggestion
    with_options if: :participation_context? do
```